internal/repository/sync.go

```diff
@@ -45,7 +49,7 @@ func Clone(ctx context.Context, remoteURL, dir, initialBranch string) (*Reposito
 		return nil, errors.Join(ErrCloningRepository, err)
 	}
 
-	if err := os.Mkdir(dir, 0755); err != nil {
+	if err := os.Mkdir(dir, 0o755); err != nil {
```

Comment:
unrelated change

Reply:
Fixed, my gofmt/goimports keeps on formating it like that.